src/redux/reducers/componentConfig/initialState.js

```diff
@@ -103,7 +105,7 @@ const heatmapInitialConfig = {
   selectedTracks: ['louvain'],
   groupedTracks: ['sample', 'louvain'],
   expressionValue: 'raw',
-  truncatedValues: false,
```

Comment:
why this change?

Reply:
because I asked @ogibson-biomage what the default value for heatmap was supposed to be and he told me truncated